src/app/config-handler.ts

```diff
@@ -283,6 +284,27 @@ class Config {
         await this.updateUserConfig({ buildNumber, installVariant: this.installVariant });
     }
 
+    /**
```

Comment:
disableGPU, is user setting in the menu.

If that value is written to user settings, it will be overwritten at every startup.
Is that the case ?

Same goes enableRendererLogs.

What is the priority of the 3 different configs ?
cloud > global > user

We need to think at every parameter, which config has priority.

As user if I do change I expect it to over rule the cloud and global

Reply:
agree, i think it is worth clarifying priorities everywhere they are used, by comment (like: this overrides the param defined in that place, and is overriden by a param defined at this other place)

Reply:
@johankwarnmarksymphony Removed disableGpu and enableRendererLogs. Rest of the fields are controlled by admins.

None of the user related fields will be overwritten now.

And, priority is Cloud > User > Global

@mattias-symphony I've added a code comment.